hq/app/logging/LogConfig.scala

```diff
@@ -31,24 +32,25 @@ object LogConfig {
     )).toString()
   }
 
-  private def buildCredentialsProvider(stsRole: String) = {
-    val random = new SecureRandom()
-    val sessionId = s"session${random.nextDouble()}"
+  private def buildCredentialsProvider(stsRole: String, config: LoggingConfig) = {
+    val sessionId = s"${config.app}-session"
 
     val instanceProvider = InstanceProfileCredentialsProvider.getInstance
     val stsClient = AWSSecurityTokenServiceClientBuilder.standard.withCredentials(instanceProvider).build
     new STSAssumeRoleSessionCredentialsProvider.Builder(stsRole, sessionId).withStsClient(stsClient).build
   }
 
   def initLocalLogShipping(config: LoggingConfig): Unit = {
-    if(config.isDev && config.localLogShippingEnabled) {
+    if(config.stage == DEV && config.localLogShippingEnabled) {
```

Comment:
This is a super minor point, but would it be helpful to log something in an `else` statement for this `if`? The remote logging says words to the effect of "I am disabled in DEV", in DEV. Should we be logging "this is only enabled in dev" and/or "local log shipping is disabled" to help people that come to this later?